Xamarin.Forms.Core.UnitTests/NavigationUnitTest.cs

```diff
@@ -96,10 +112,15 @@ public class NavigationUnitTest : BaseTestFixture
 			bool fired = false;
 			nav.Pushed += (sender, e) => fired = true;
 
+			Assert.AreEqual(childRoot, nav.RootPage);
+			Assert.AreEqual(childRoot, nav.CurrentPage);
+
 			await nav.PushAsync (childRoot);
 			
 			Assert.False (fired);
-			Assert.AreEqual (childRoot, nav.CurrentPage);
+			Assert.AreEqual(childRoot, nav.RootPage);
```

Comment:
I can't remember why this was the case since it's been a while. I think I had copied the test from an existing one as a template.